webpack/assets/javascripts/react_app/pages/common/PageLayout/PageLayout.js

```diff
@@ -61,7 +66,29 @@ const PageLayout = ({
             )}
           </Col>
           <Col id="title_action" md={searchable ? 6 : 8}>
-            <div className="btn-toolbar pull-right">{toolbarButtons}</div>
+            <div className="btn-toolbar pull-right">
+              <Slot multi id="layout-toolbar-button" />
```

Comment:
the id is not specific to a domain, i.e - a plugin wants to add a button only to the host index page, and not in all pages.